common/src/main/res/values/strings.xml

```diff
@@ -35,4 +35,7 @@
 
     <!-- Design system UI elements -->
     <string name="search_hint">Search</string>
+
+    <string name="CoinBase" translatable="false">CoinBase</string>
+    <string name="coinbase_fiat_balance_with_currency" translatable="false">=%s %s</string>
```

Comment:
Maybe these should go in the coinbase module.

Reply:
Also, this string `%s %s` may already exist.

Reply:
Should `CoinBase` be `Coinbase`? Can we just use what is in the `coinbase_integration` module instead?